ui/admin/app/components/form/target/add-credential-libraries/index.js

```diff
@@ -26,12 +26,12 @@ export default class FormTargetAddCredentialLibrariesIndexComponent extends Comp
    * @type {[CredentialLibraryModel]}
    */
   @computed(
-    'args.{credentialLibraries.[],model.application_credential_library_ids.[]}'
+    'args.{credentialLibraries.[],model.application_credential_source_ids.[]}'
   )
   get filteredCredentialLibraries() {
     // Get IDs for credential libraries already added to the current target
     const currentCredentialLibraryIDs =
-      this.args.model.application_credential_library_ids.map(
+      this.args.model.application_credential_source_ids.map(
         (credentialLibrary) => credentialLibrary.value
       );
     const notAddedCredentialLibraries = this.args.credentialLibraries.filter(
```

Comment:
const notAddedCredentialLibraries = this.args.credentialLibraries.filter --> const notAddedCredentialSources = ...